utils/nfc-mfultralight.c

```diff
@@ -388,19 +393,23 @@ write_card(bool write_otp, bool write_lock, bool write_uid)
     }
   }
 
-  // NTAG Dynamic Lock Bytes are in different locations for each type
   for (uint32_t page = uiSkippedPages; page < uiBlocks; page++) {
-    if (((iNTAGType == NTAG_NONE && page == 0x2) || \
-         (iNTAGType == NTAG_213 && page == 0x28) || \
-         (iNTAGType == NTAG_215 && page == 0x82) || \
-         (iNTAGType == NTAG_216 && page == 0xe2)) && (!write_lock)) {
+    if ((!write_lock) && page == 0x2) {
+      printf("s");
+      uiSkippedPages++;
+      continue;
+    }
+    // NTAG doesn't have OTP blocks
```

Comment:
Scoop: yes NTAG also have OTP :) Capability container at page 0x02 is just another name but still with OTP.

Reply:
Ah, I knew I should have checked! :P  OK, I'll tweak it (again!). Lol.